v1/pipeline/steps/cd/helm-canary-step-info.yaml

```diff
@@ -33,6 +33,14 @@ allOf:
         - type: string
           pattern: (<\+.+>.*)
           minLength: 1
+      commandFlags:
+        oneOf:
+        - type: array
+          items:
+            $ref: helm-manifest-command-flag.yaml
+        - type: string
+          pattern: ^<\+input>((\.)((executionInput\(\))|(allowedValues|selectOneFrom|default|regex)\(.+?\)))*$
```

Comment:
Is this pattern something that we usually use? Looks to me that would be very troublesome if we plan that a new option in the runtime input settings

Reply:
I see it a few times in our code, including the file [k8s-apply-step-info.yaml](https://github.com/harness/harness-schema/blob/main/v0/pipeline/steps/cd/k8s-apply-step-info.yaml) which I used as a reference to make the same change.

I got your point, agree, and hope we have a plan for that.

Reply:
I will follow this recommendation on the next schema change.